### ISSUE TITLE
Button have issue in Microsoft Edge

### ISSUE THREAD

**Issue by BigEgg** [angular/material#8575]
**Actual behavior**:
-  What is the issue? *

The first time click the button, all animation and click function are work fine. but without focus on other element, when click the button again, there has no animation and click function not been triggered.
-  What is the expected behavior?

No matter how many times to click that button, should behavior the same as first time

**CodePen** or Steps to reproduce the issue: *
-  CodePen Demo which shows your issue :

No need for CodePen, demo site can reproduce this issue.
-  Details:

The first button on [button page](https://material.angularjs.org/latest/demo/button);

and the "Toggle left" button on [Sidenav page](https://material.angularjs.org/latest/demo/sidenav)

**Angular Versions**: *
-  Angular Version:

Same as Demo site
-  Angular Material Version:

Same as Demo site (currently 1.0.9).

**Additional Information**:
-  Browser Type: *

Microsoft Edge
-  Browser Version: *

   - Microsoft Edge 25.10586.0.0
   - Microsoft EdgeHTML 13.10586

-  OS: *

   WIndows 10
-  Stack Traces:

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by ThomasBurleson** [comment]
@crisbeto  - can you investigate ?

**Comment by BigEgg** [comment]
Any update?

Just find that this had been mark as `IE`. But in fact, `IE` work fine with this case. Only `Microsoft Edge` happened.

Also just find out new demo site had add `md-disable-backdrop` for left sidenav, if remove that, 1.1.0-RC5 still have this issue.

**Comment by crisbeto** [comment]
Well I did look into it a few days ago and managed to reproduce it occasionally, but I couldn't find anything obvious that might be causing it.

**Comment by BigEgg** [comment]
No, click not fired.

You can use these step to reproduce this issue, in the demo site:
> Please use [1.0.9 version demo site](https://material.angularjs.org/1.0.9/demo/sidenav), since the [new version](https://material.angularjs.org/HEAD/demo/sidenav) add `md-disable-backdrop` in the left SideNav

* Open the demo site in *Microsoft Edge*
* Resize the page to show `Toggle Left` _button_
* Click the `Toggle Left` _Button_, and the left _SideNav_ opened correctly
* Click the gray out background to toggle _SideNav_ close

At this point, you can see:

* the ink of the `Toggle Left` _Button_ still there
* and when click that button, left SideNav will never open again, unless, 
  * you click the empty space
  * or click the `Toggle Right` button

As you can see from the screen shot, "toggle left is done" will never logged until that button lost focus. Seem like the BackDrop of the SideNav block button's Blur function or what else.

![image](https://cloud.githubusercontent.com/assets/1243699/15990578/f1334a14-30c8-11e6-99bc-a2160db5f454.png)

**Comment by crisbeto** [comment]
Hmm ok good to know, that wasn't what I understood from the issue description. Thanks for the explanation.

**Comment by crisbeto** [comment]
I tried looking into it again and I can reproduce it locally, however I think that this might be a browser issue since I couldn't find anything in Material that could be causing it. All the sidenav does is get the `activeElement` when it gets opened and then focusing it when the sidenav gets closed. I also tried checking the `mdButton` directive, but it doesn't do anything in particular that might cause this. I event tried replacing the button with something like a `div` or a `h1`, but the problem is still there.

I'll close this, because it looks like our hands are tied, but it can always be re-opened if some new info shows up.
